app/src/main/res/values/strings.xml

```diff
@@ -229,4 +229,5 @@
     <string name="tt_yim_charts_parent">tt_yim_charts_parent</string>
     <string name="tt_yim_statistics_parent">tt_yim_statistics_parent</string>
     <string name="title_activity_settings">SettingsActivity</string>
+    <string name="cb_review_tc">You acknowledge and agree that your contributed reviews to CritiqueBrainz are licensed under a Creative Commons Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0) license. You agree to license your work under this license. You represent and warrant that you own or control all rights in and to the work, that nothing in the work infringes the rights of any third-party, and that you have the permission to use and to license the work under the selected Creative Commons license. Finally, you give the MetaBrainz Foundation permission to license this content for commercial use outside of Creative Commons licenses in order to support the operations of the organization.</string>
```

Comment:
Why are we still using the strings from res? Just add these texts to the Constants file

Reply:
IDE recommendation. Plus this is a long text and standalone objects acquire runtime memory and persists application lifecycle.

Reply:
Given that this isn't going to be frequently used as much. I think it should belong here.

Reply:
I wanna get away from these stupid config files that android gives us. Let the devs create their own files and follow the process their way, I feel. But for now, let's let it be then. Also, the memory reason isn't good enough. We want better code readability. Memory consumption for such text is really irrelevant

Reply:
I think I might know of a way